dao/cluster_dao_impl.go

```diff
@@ -223,6 +224,31 @@ func (c *ClusterDaoImplCassandra) GetAllEntitiesInfo() []e.EntityInfo {
 	return entities
 }
 
+func (c *ClusterDaoImplCassandra) GetAllEntitiesForApp(id string) []e.EntityInfo {
+	var nodeName string
+	var status int
+	var history string
+
+	var entities []e.EntityInfo
+	query := fmt.Sprintf(KeyGetAllEntitiesForApp, id)
+	iter := c.Session.
+		Query(query).
+		Consistency(c.Conf.ClusterDB.DBConfig.Consistency).
+		PageSize(c.Conf.ClusterDB.DBConfig.PageSize).
+		Iter()
+
+	for iter.Scan(&id, &nodeName, &status, &history) {
+		entities = append(entities, e.EntityInfo{Id: id, Node: nodeName, Status: status, History: history})
+	}
+
+	if err := iter.Close(); err != nil {
+		glog.Fatal(err)
```

Comment:
@prakhar2611 Let's not use fatal here. If it fails lets log it and it is fine. Fatal should be used in case we can overcome from the error.

Reply:
sure, I have remove the fatal and returned the error instead.